invokeai/frontend/web/src/app/store/middleware/listenerMiddleware/listeners/modelsLoaded.ts

```diff
@@ -164,7 +164,7 @@ const handleVAEModels: ModelHandler = (models, state, dispatch, log) => {
   // We have a VAE selected, need to check if it is available
 
   // Grab just the VAE models
-  const vaeModels = models.filter(isNonFluxVAEModelConfig);
+  const vaeModels = models.filter((m) => isNonFluxVAEModelConfig(m));
```

Comment:
Without this, JS uses the index as second arg automatically and causes a type error since the second arg for this is an optional boolean
<img width="681" alt="Screenshot 2024-11-04 at 8 18 45 AM" src="https://github.com/user-attachments/assets/9ff94142-cf93-405c-b6d9-721adf8db834">